### ISSUE TITLE
Investigate failing acceptance tests 20190621

### ISSUE THREAD

**Issue by phil-davis** [owncloud/files_primary_s3#223]
https://drone.owncloud.com/owncloud/files_primary_s3/989/1119 and
https://drone.owncloud.com/owncloud/files_primary_s3/989/1404
```
--- Failed scenarios:

    /drone/server/tests/acceptance/features/apiMain/checksums.feature:287
    /drone/server/tests/acceptance/features/apiMain/checksums.feature:288
    /drone/server/tests/acceptance/features/apiMain/checksums.feature:299

82 scenarios (79 passed, 3 failed)
```

```
  Scenario Outline: Uploading a file with invalid SHA1 checksum overwriting an existing file                                                                                                     # /drone/server/tests/acceptance/features/apiMain/checksums.feature:280
    Given using <dav_version> DAV path                                                                                                                                                           # FeatureContext::usingOldOrNewDavPath()
    When user "user0" uploads file "filesForUpload/textfile.txt" to "/textfile0.txt" with checksum "SHA1:f005ba11f005ba11f005ba11f005ba11f005ba11" using the WebDAV API                          # ChecksumContext::userUploadsFileToWithChecksumUsingTheAPI()
    Then as user "user0" the webdav checksum of "/textfile0.txt" via propfind should match "SHA1:0c1d334e686d1039c9ead0dbc047f02dbf696be8 MD5:d991cd854c53729d066c6ed5e34bcda3 ADLER32:8685092b" # ChecksumContext::theWebdavChecksumOfViaPropfindShouldMatch()
    And the content of file "/textfile0.txt" for user "user0" should be "ownCloud test text file 0" plus end-of-line                                                                             # FeatureContext::contentOfFileForUserShouldBePlusEndOfLine()

    Examples:
      | dav_version |
      | old         |
        Warning: Illegal string offset 'value' in /drone/server/tests/acceptance/features/bootstrap/ChecksumContext.php line 140
      | new         |
        Warning: Illegal string offset 'value' in /drone/server/tests/acceptance/features/bootstrap/ChecksumContext.php line 140
```

```
  Scenario: Upload overwriting a file with new chunking and invalid checksum                                                           # /drone/server/tests/acceptance/features/apiMain/checksums.feature:299
    Given using new DAV path                                                                                                           # FeatureContext::usingOldOrNewDavPath()
    When user "user0" creates a new chunking upload with id "chunking-42" using the WebDAV API                                         # FeatureContext::userCreatesANewChunkingUploadWithId()
    And user "user0" uploads new chunk file "2" with "BBBBB" to id "chunking-42" using the WebDAV API                                  # FeatureContext::userUploadsNewChunkFileOfWithToId()
    And user "user0" uploads new chunk file "3" with "CCCCC" to id "chunking-42" using the WebDAV API                                  # FeatureContext::userUploadsNewChunkFileOfWithToId()
    And user "user0" moves new chunk file with id "chunking-42" to "/textfile0.txt" with checksum "SHA1:f005ba11" using the WebDAV API # FeatureContext::userMovesNewChunkFileWithIdToMychunkedfileWithChecksum()
    Then the HTTP status code should be "400"                                                                                          # FeatureContext::theHTTPStatusCodeShouldBe()
    And the content of file "/textfile0.txt" for user "user0" should be "ownCloud test text file 0" plus end-of-line                   # FeatureContext::contentOfFileForUserShouldBePlusEndOfLine()
      Failed asserting that two strings are equal.
      --- Expected
      +++ Actual
      @@ @@
      -'ownCloud test text file 0\n
      +'<?xml version="1.0" encoding="utf-8"?>\n
      +<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">\n
      +  <s:exception>Sabre\DAV\Exception\NotFound</s:exception>\n
      +  <s:message>File with name textfile0.txt could not be located</s:message>\n
      +</d:error>\n
```

- [ ] fix the `Warning: Illegal string offset` so that the test reports more nicely when something is missing
- [ ] investigate if this is a real/new problem introduced by some recent core change, or a test issue, or?

**Comment by phil-davis** [comment]
@dpakach I assigned you, since you skipped the tests again last week.
Is there anything more that QA can do here?
Or is it "over to the developers" to fix https://github.com/owncloud/files_primary_s3/issues/224 ?

**Comment by dpakach** [comment]
> @dpakach I assigned you, since you skipped the tests again last week.
> Is there anything more that QA can do here?
> Or is it "over to the developers" to fix #224 ?

@phil-davis I dont think there is any thing we can do here. The remaining tests have passed (except for some phpUnit intermettent failure). It is just a small bug that needs to be fixed. So I think we can close this one.
